### PR TITLE
changes in service provider

### PR DESCRIPTION
**Pull request by Harikrishnan1819**
created condition for check env variables

### CODE REVIEW

File: src/BugAtlasServiceProvider.php [hunk 1]
```diff
@@ -21,8 +21,14 @@ class BugAtlasServiceProvider extends ServiceProvider
     public function boot(Request $request)
     {
         $this->mergeConfigFrom(__DIR__ . '/../config/bugatlas.php', 'bugatlas');
-        $logDetails = $this->prepareLogDetails($request);
-        $this->sendLogToApi($logDetails);
+
+        // Check if the required keys are available in the merged configuration
```
Comment (prasanth-spark) [inline src/BugAtlasServiceProvider.php]: Remove the comments and add inn multi-line comments.

File: src/BugAtlasServiceProvider.php [hunk 2]
```diff
@@ -21,8 +21,14 @@ class BugAtlasServiceProvider extends ServiceProvider
...
+        if (!empty(config('bugatlas.api_key')) && !empty(config('bugatlas.secret_key')) && !empty(config('bugatlas.tag'))) {
```
Comment (prasanth-spark) [inline src/BugAtlasServiceProvider.php]: Change the logic to check the values are empty, if empty return false.

File: src/BugAtlasServiceProvider.php (hunk 1 above)
Comment (Harikrishnan1819) [reply src/BugAtlasServiceProvider.php]: coments resolved

File: src/BugAtlasServiceProvider.php (hunk 2 above)
Comment (Harikrishnan1819) [reply src/BugAtlasServiceProvider.php]: comments resolved

**Pull request merged**